### ISSUE TITLE
doinstall script fails

### ISSUE THREAD

**Issue by evanescente-ondine** [AdaCore/gtkada#15]
I can't execute the doinstall script properly. It runs under root, and there's the last GNAT in PATH. So it should not happen.
```
checking that your gnat compiler works with a simple example... gnatbind: Cannot find: s-stalib.ali
gprbind: invocation of gnatbind failed
gprbuild: unable to bind conftest.adb
no 
configure: error: /usr/bin/gprbuild test failed at compile time! Check your configuration.
```

**Comment by setton** [comment]
I think this is a compiler installation problem... Can you build a simple "hello world" program with that compiler?

**Comment by evanescente-ondine** [comment]
I can.

**Comment by evanescente-ondine** [comment]
here's the content of install.log
gnatprep -DGETTEXT_INTL=False -DHAVE_GETTEXT=True src/gtkada-intl.gpb src/gtkada-intl.adb
====== Building static-pic libraries =====
/usr/bin/gprbuild  -j0 -m -p  -XLIBRARY_TYPE=static-pic -Psrc/gtkada.gpr
object directory "/home/mehdi/Téléchargements/gtkada-gpl-2017-x86_64-linux-bin/src/obj/gtkada/static-pic/" created
library directory "/home/mehdi/Téléchargements/gtkada-gpl-2017-x86_64-linux-bin/src/lib/gtkada/static-pic" created for project gtkada
gtkada.gpr:38:25: illegal value for Library_Kind
gprbuild: "src/gtkada.gpr" processing failed
Makefile:84 : la recette pour la cible « build_library_type/static-pic » a échouée
make: *** [build_library_type/static-pic] Erreur 4

**Comment by evanescente-ondine** [comment]
Still the same thing. ./doinstall gives
checking that your gnat compiler works with a simple example... entity configurations_vx653:5:6: Undefined entity 'config_wind_base'
gprbuild: could not parse the XML files in /usr/local/share/gprconfig
no configure: error: /usr/local/bin/gprbuild test failed at compile time! Check your configuration.
An error occurred. Please see install.log.

But there's no install.log in the compilation directory this time.

**Comment by setton** [comment]
do you have a "LIBRARY_KIND" defined in the environment?

**Comment by evanescente-ondine** [comment]
No... But nothing of this kind was mentioned in the INSTALL file. Or
did I miss ?

2018-01-15 18:06 UTC+01:00, Nicolas Setton <notifications@github.com>:

**Comment by hectoruhalte** [comment]
I am having the exact same problem (gtkada-community-2018-20180523-x86_64-linux-bin.tar.gz version)
I have set the Library_Kind variable to static-pic or relocatable, but it does not seem to work.
I have got the same error when building static-pic libraries.

====== Building static-pic libraries =====
/usr/bin/gprbuild  -j0 -m -p  -XLIBRARY_TYPE=static-pic -Psrc/gtkada.gpr
gtkada.gpr:38:25: illegal value for Library_Kind
gprbuild: "src/gtkada.gpr" processing failed
